apps/web/pages/signup.tsx

```diff
@@ -461,54 +468,59 @@ export default function Signup({
             </div>
           </div>
         </div>
-        <div
-          className="border-subtle hidden w-full flex-col justify-between rounded-l-2xl border py-12 pl-12 lg:flex"
-          style={{
-            background:
-              "radial-gradient(162.05% 170% at 109.58% 35%, rgba(102, 117, 147, 0.7) 0%, rgba(212, 212, 213, 0.4) 100%) ",
-          }}>
-          {IS_CALCOM && (
-            <div className="mb-12 mr-12 grid h-full w-full grid-cols-4 gap-4 ">
-              <div className="">
-                <img src="/product-cards/trustpilot.svg" className="h-[54px] w-full" alt="#" />
-              </div>
-              <div>
-                <img src="/product-cards/g2.svg" className="h-[54px] w-full" alt="#" />
-              </div>
-              <div>
-                <img src="/product-cards/producthunt.svg" className="h-[54px] w-full" alt="#" />
-              </div>
+        <div className="border-muted mx-auto mt-24 w-full max-w-2xl flex-col justify-between rounded-l-2xl bg-[radial-gradient(162.05%_170%_at_109.58%_35%,_rgba(102,_117,_147,_0.7)_0%,_rgba(212,_212,_213,_0.4)_100%)] pl-4 dark:bg-none lg:mt-0 lg:flex lg:max-w-full lg:border lg:py-12 lg:pl-12">
+          <div className="mb-12 mr-12 grid w-full grid-cols-3 gap-3 lg:grid-cols-4">
+            <div>
+              <img
+                src="/product-cards/trustpilot.svg"
+                className="block h-[54px] w-full dark:hidden"
+                alt="Trustpilot Rating of 4.7 Stars"
+              />
+              <img
+                src="/product-cards/trustpilot-dark.svg"
+                className="hidden h-[54px] w-full dark:block"
+                alt="Trustpilot Rating of 4.7 Stars"
+              />
             </div>
-          )}
+            <div>
+              <img src="/product-cards/g2.svg" className="h-[54px] w-full" alt="G2 Rating of 4.7 Stars" />
+            </div>
+            <div>
+              <img
+                src="/product-cards/producthunt.svg"
+                className="h-[54px] w-full"
+                alt="G2 Rating of 4.7 Stars"
+              />
+            </div>
+          </div>
           <div
-            className="border-default rounded-bl-2xl rounded-br-none rounded-tl-2xl border-dashed py-[6px] pl-[6px]"
+            className="border-default hidden rounded-bl-2xl rounded-br-none rounded-tl-2xl border-dashed lg:block lg:py-[6px] lg:pl-[6px]"
             style={{
               backgroundColor: "rgba(236,237,239,0.9)",
             }}>
-            <img src="/mock-event-type-list.svg" alt="#" className="" />
+            <img src="/mock-event-type-list.svg" alt="Cal.com Booking Page" />
```

Comment:
Does this need white labeling with APP_NAME?

Reply:
right.. lemme remove cal.com